WooCommerce/Classes/ViewRelated/Notifications/NotificationsViewController.swift

```diff
@@ -436,26 +355,6 @@ private extension NotificationsViewController {
     /// Refreshes the Results Controller Predicate, and ensures the UI is in Sync.
     ///
     func reloadResultsController() {
-        refreshResultsPredicate()
-        tableView.reloadData()
-    }
-
-    func refreshResultsPredicate() {
-        let notDeletedPredicate = NSPredicate(format: "deleteInProgress == NO")
```

Comment:
this `notDeletedPredicate` was used in combination with the subtype predicate, do you know if we might still want to include this filter on deleted notifications?

Reply:
ah, nice catch, thanks!